rel/ci.vm.args

```diff
@@ -21,7 +21,7 @@
 ##-heart
 
 # GC more often
--env ERL_FULLSWEEP_AFTER 10
+# -env ERL_FULLSWEEP_AFTER 10
```

Comment:
@lazedo lets remove the line rather than comment out.  Git has a history feature ;)

Reply:
pushed in #5628 & #5629